src/engraving/libmscore/line.cpp

```diff
@@ -1498,7 +1498,11 @@ bool SLine::setProperty(Pid id, const PropertyValue& v)
         _lineColor = v.value<mu::draw::Color>();
         break;
     case Pid::LINE_WIDTH:
-        _lineWidth = v.value<Millimetre>();
+        if (v.type() != P_TYPE::SPATIUM) {
```

Comment:
It should be v.type() == P_TYPE::MILLIMETRE

Reply:
done, thanks 👍